contracts/script/deploy/certverifier/README.md

```diff
@@ -1,6 +1,6 @@
 ## EigenDA Cert Verfier Deployer
 
-This script can be used to deploy an EigenDACertVerifier contract with custom security thresholds and quorum numbers. The deployment should only be performed on Ethereum L1 testnet or mainnet environment and is not supported on L2s.
+This script can be used to deploy an EigenDACertVerifierV2 contract with custom security thresholds and quorum numbers. The deployment should only be performed on Ethereum L1 testnet or mainnet environment and is not supported on L2s.
```

Comment:
You added a new EigenDACertVerifierV1Deploy in this directory, why now also mention its usage in the README? Shouldn't we say that you can deploy both?

Reply:
Btw do we also have another deployer for the new CertVerifier (currently v3)?

Reply:
I think ethen has another PR for that.